samples/serialization/entropy_nrf53/cpunet/prj.conf

```diff
@@ -0,0 +1,17 @@
+CONFIG_ENTROPY_GENERATOR=y
+
+CONFIG_IPM=y
+CONFIG_IPM_NRFX=y
+
+CONFIG_IPM_MSG_CH_1_ENABLE=y
+CONFIG_IPM_MSG_CH_1_RX=y
+CONFIG_IPM_MSG_CH_0_ENABLE=y
+CONFIG_IPM_MSG_CH_0_TX=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+CONFIG_OPENAMP=y
+CONFIG_TINYCBOR=y
+
+CONFIG_RP_SERIALIZATION=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
```

Comment:
This looks odd. This will prevent the linker from removing unused code, which may increase the final binary size a lot

Reply:
FYI; the conclusion is correct.

But the argument is wrong.

You are probably thinking about --no-gc-sections, which will control whether or not unused sections should be discarded or not.

Orphan handling is about what to do with sections that are definitely used, but have section names that are not recognized (matched) by the LD script. The options are to error-out, place and warn, or place without a warning.

Reply:
@SebastianBoe , yes you're right. Thanks for the explanation

Reply:
Hold on, surely the right solution to the missing linker section issue is to add a linker section, like @SebastianBoe explained [here](https://github.com/NordicPlayground/nrfxlib/pull/149#discussion_r394440192), not change the linker behavior for all applications that want to use this module?

Reply:
Here's an Sebastian's suggested mechanism in use in some module:
- [Linker script snippet](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/subsys/bluetooth/mesh/sensor_types.ld)
- [CMakeFile](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/subsys/bluetooth/mesh/CMakeLists.txt#L44)

Reply:
Correct, there is already agreement that we can't use CONFIG_LINKER_ORPHAN_SECTION_PLACE=y

Reply:
Yes `CONFIG_LINKER_ORPHAN_SECTION_PLACE` is removed